data-raw/process_data_sets.R

```diff
@@ -272,3 +272,17 @@ pennies_resamples <-
   select(replicate, everything()) %>%
   unnest(cols = c(data))
 usethis::use_data(pennies_resamples, overwrite = TRUE)
+
+
+
+#-------------------------------------------------------------------------------
+# Coffee quality data
+# Google Sheet here: https://docs.google.com/spreadsheets/d/1fscb1AbsSXWqqws-hhWAZfT580ms_FvOkBBQsJwozSY/edit
+# Original Github source: https://github.com/jldbc/coffee-quality-database
+#-------------------------------------------------------------------------------
+
+coffee_ratings <- 
+  "https://wjhopper.github.io/SDS-201/data/coffee_ratings.csv" %>%
+  readr::read_csv()
+
+usethis::use_data(coffee_ratings, overwrite = TRUE)
```

Comment:
If your dataset is called `coffee_ratings`, please delete the `coffee_quality.rda` file from the `data` folder.

Reply:
Addressed in commit [4ecc0b6](https://github.com/beanumber/moderndive/pull/13/commits/4ecc0b6c1df98b66afa6cb26df3585e522305c99)